stable/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 5.8.0
+version: 5.8.1
```

Comment:
Increase minor version: `5.9.0`

Reply:
Hi @juan131, did you check the latest commit with all the fixed?